src/core/server/elasticsearch/client/configure_client.ts

```diff
@@ -98,10 +98,16 @@ function getResponseMessage(event: RequestEvent): string {
 const addLogging = (client: Client, logger: Logger) => {
   client.on('response', (error, event) => {
     if (event) {
+      const opaqueId = event.meta.request.options.opaqueId;
+      const meta = opaqueId
+        ? {
+            http: { request: { id: event.meta.request.options.opaqueId } },
```

Comment:
we will remove this logic when we start including `trace.id` in the logs. https://github.com/elastic/kibana/issues/102699 
Instead, `trace.id` will be used for the log correlation. But right now we need it for testing.